src/graphchain/core.py

```diff
@@ -44,6 +45,11 @@ def layer_setitem(self: Layer, key: Hashable, value: Any) -> None:
 logger = logging.getLogger(__name__)
 
 
+def joblib_dump_lz4(obj: Any, fd: io.BytesIO) -> None:
```

Comment:
Could this allow for a customizable compression, and then you also export a partial version of this which is the default? ie `joblib_dump_lz4 = partial(joblib_dump, compression="lz4")`

Reply:
Done. The default implementation allows for customization as follows:
```
joblib_dump_gzip = partial(joblib_dump, compress="gzip", ext="joblib.gz", protocol=HIGHEST_PROTOCOL)
joblib_load_gzip = partial(joblib_load, ext="joblib.gz")
```